src/internal/common/dttm/dttm_test.go

```diff
@@ -58,6 +58,49 @@ func (suite *DTTMUnitSuite) TestParseTime() {
 	require.Error(t, err, clues.ToCore(err))
 }
 
+func (suite *DTTMUnitSuite) TestOrNow() {
+	var (
+		oneMinuteAgo = time.Now().Add(-1 * time.Minute)
+		inOneMinute  = time.Now().Add(1 * time.Minute)
+		inOneHour    = time.Now().Add(1 * time.Hour)
```

Comment:
nit: there's no constant multiple required if you're just adding 1 of each

Reply:
I know, it's just a standard pattern for me.